web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -489,77 +494,124 @@ describe("Formation - BusinessStep", () => {
       });
     }
 
-    describe("Business Designator Options based on Will Practice Law Answer", () => {
-      it.each(corpLegalStructures)(
-        "Shows PA and PC options for Business Designator when Will You Practice Law is Yes",
-        async (legalStructureId) => {
-          await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, { willPracticeLaw: true });
+    describe("Business Designator", () => {
+      describe("Business Designator secondary label foreign corporation", () => {
+        it.each(corpLegalStructures)(
+          "Shows secondary label foreign corporation",
+          async (legalStructureId) => {
+            await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, {});
+            expect(
+              screen.getByText(Config.formation.fields.businessSuffix.labelSecondaryTextForeignCorporation)
+            ).toBeInTheDocument();
+            expect(
+              screen.getByText(Config.formation.fields.businessSuffix.labelSecondaryTextForeignCorporation)
+            ).toBeInTheDocument();
+          }
+        );
 
-          expect(screen.queryByText("P.C.")).not.toBeInTheDocument();
-          expect(screen.queryByText("P.A.")).not.toBeInTheDocument();
+        it.each(nonCorporationLegalStructures)(
+          "Does Not shows secondary label foreign corporation",
```

Comment:
Can you name this more specifically? Not sure why it shouldn't appear from title

Reply:
renamed